### ISSUE TITLE
Program Overview Page

### ISSUE THREAD

**Issue by bentvelj** [dimitritsampiras/olympian#105]
We have a few steps to create a program, but this is only the start of the process. A program contains workouts, and a workout contains exercises. After submitting some program tags, the user should be presented with a page that gives them the option to add a workout. 

Should probably work with @dimitritsampiras to get some Figma mocks since the existing ones are outdated, and are not really in line with current plans. The page should display all the information that the user has entered thus far (name, focus, publicity, tags). Each time the user adds a workout, it should be displayed on this page in the form of a card or something, which I can then tap to view the workout details.

For this task, use mock data for the cards for now, and tapping an existing card doesn't have to go anywhere. We can link into workout creation (and subsequently exercise creation) afterwards.

Bottom line is that I should be able to have an overview of my program and the workouts involved.

Here's a low fidelity protoype of what I'm envisioning
![image](https://user-images.githubusercontent.com/45087499/204401771-bdcb8619-a6be-4f11-8839-ab8cd2dab390.png)

Design and implementation is up to you

**Comment by dimitritsampiras** [comment]
working on this now
